client/src/modules/cohort/components/CohortHeader.jsx

```diff
@@ -6,6 +6,7 @@ import { connect } from 'react-redux';
 import { CohortIcon } from 'assets/images/icons';
 import { updateCohort } from '../model/actions';
 import { RouterMatchPropType } from 'common/constants/propTypes';
+import { whiteSpaceOnly } from 'common/utils/helpers';
```

Comment:
space is always "white" character, if there would be more white characters, this method could be named whiteCharsOnly, but in this case maybe it's better to change it's name to spacesOnly?

Reply:
I meant space as `przestrzeń` not as space key. 

https://developer.mozilla.org/pl/docs/Web/CSS/white-space

white space means basically all white characters